WordPress/gutenberg
# [RNMobile] Remove remaining wpcom references after moving gutenberg-mobile code

## Description


This PR removes remaining Automattic references in iOS related files and wpcom related supported path in `apiFetch`. 
Seems the `apiFetch` path pattern was introduced in gutenberg-mobile as a trial and now it shouldn't be needed: https://github.com/wordpress-mobile/gutenberg-mobile/commit/d1c4d905e54407c6511c9b0d3cd0d023856dd55c

gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2562

## How has this been tested?




Use main app PRs: 
- https://github.com/wordpress-mobile/WordPress-iOS/pull/14684
- https://github.com/wordpress-mobile/WordPress-Android/pull/12722

1. Open the editor
1. Add an image block
1. Upload an image
1. Check if it's uploaded without any errors

## Screenshots 

## Types of changes



Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
